web/ui/react-app/src/pages/targets/JobHeader.module.css

```diff
@@ -0,0 +1,22 @@
+.title {
+  font-size: 20px;
+  font-weight: bold;
+  cursor: pointer;
+}
+
+.normal {
+  composes: title;
+}
+
+.danger {
+  composes: title;
+  color: rgb(242, 65, 65);
+}
+
+.expand-btn {
```

Comment:
I guess all ```!important``` are needed to override default button styles, right?

Reply:
@boyskila I think so. It seems that if not, the default styles get applied... unless you know of another way? I'm all ears!

Reply:
yes, inline styles will override the default styles unless they are not marked as ```!important``` as well. ```!important``` is dangerous and I try to avoid it if it's possible

Reply:
okay i went ahead and removed all the `!important`s. Thanks for the insight @boyskila !